docs/e2e.md

```diff
@@ -0,0 +1,35 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Running E2E Tests](#running-e2e-tests)
+    - [Reference](#reference)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Running E2E Tests
+
+1. Ensure **operator-sdk** is installed.
```

Comment:
kubeconfig also need ready

Reply:
I notify users to `login to your OpenShift cluster as an admin user` .

Reply:
👌